### PR TITLE
[INLONG-9041][Manager] Support saving schema information when saving iceberg source

### LINKED ISSUE #9041: [Improve][Manager] Support saving schema information when saving iceberg source
### Description

Support saving schema information when saving iceberg source.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #9041 

### Motivation

Support saving schema information when saving iceberg source.

### Modifications

Support saving schema information when saving iceberg source.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceOperator.java [hunk 1]
```diff
@@ -116,4 +116,11 @@ default Map<String, List<StreamSource>> getSourcesMap(InlongGroupInfo groupInfo,
      */
     void restartOpt(SourceRequest request, String operator);
 
+    /**
+     * Get the source field info.
+     *
+     * @param request request of source
```
Comment (EMsnap) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceOperator.java:122]: typo here and missing parameter

**Review by vernedeng — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/iceberg/IcebergSourceOperator.java [hunk 2]
```diff
@@ -83,4 +96,52 @@ public StreamSource getFromEntity(StreamSourceEntity entity) {
         source.setFieldList(sourceFields);
         return source;
     }
+
+    @Override
+    @Transactional(rollbackFor = Throwable.class, isolation = Isolation.REPEATABLE_READ)
+    public void getFieldInfo(SourceRequest request, String operator) {
```
Comment (EMsnap) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/iceberg/IcebergSourceOperator.java:102]: the method name is get field info but the actual action is update iceberg field info

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceOperator.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceOperator.java:122]: Fix.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/iceberg/IcebergSourceOperator.java (hunk 2 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/iceberg/IcebergSourceOperator.java]: Fixed.

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**